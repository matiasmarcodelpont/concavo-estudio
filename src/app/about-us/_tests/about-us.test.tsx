import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import AboutUs from '@/app/about-us/page'

describe('About Us', () => {
  it('renders About Us unchanged', () => {
    const { container } = render(<AboutUs />)
    expect(container).toMatchSnapshot()
  })
})
