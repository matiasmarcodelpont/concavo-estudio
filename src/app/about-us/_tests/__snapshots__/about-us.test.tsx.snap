// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Us renders About Us unchanged 1`] = `
<div>
  <main
    class="text-center space-y-24 pt-12"
  >
    <h1
      class="uppercase font-editorial-new normal font-light text-center text-6xl xs:text-7xl sm:text-7xl"
    >
      <span
        class="italic"
      >
        About 
      </span>
      Us
    </h1>
    <section
      class="space-y-24 sticky top-[72px] md:top-[81px] bg-bone"
    >
      <div
        class="min-h-[400px] sm:min-h-[600px] md:min-h-[800px] lg:min-h-[900px] relative"
      >
        <img
          alt="Creadores de Cóncavo"
          class="object-cover"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=3840&q=75"
          srcset="/_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Fcoti-felix.png&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
    </section>
    <div
      class="space-y-24 z-10 bg-bone relative py-16"
    >
      <section
        class="space-y-24 bg-bone"
      >
        <article
          class="flex justify-center items-center"
        >
          <p
            class="text-base xs:text-lg sm:text-xl px-8 xs:px-10 sm:px-14 max-w-[800px] text-center text-balance"
          >
            Somos un equipo conformado por la arquitecta Constanza Llorente y el artista visual Félix Llambias. Creamos y desarrollamos productos elaborados localmente para el hogar. Trabajamos con artesanos apasionados por el arte y la belleza para crear piezas únicas hechas por manos argentinas.
          </p>
        </article>
      </section>
      <section
        class="px-14 flex"
      >
        <svg
          class="text-black size-[200px] m-auto"
          fill="none"
          height="24"
          viewBox="0 0 569 569"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <mask
            height="569"
            id="mask0_2090_30"
            maskUnits="userSpaceOnUse"
            style="mask-type: alpha;"
            width="569"
            x="0"
            y="0"
          >
            <rect
              fill="currentColor"
              height="569"
              width="569"
            />
          </mask>
          <g
            mask="url(#mask0_2090_30)"
          >
            <mask
              height="356"
              id="mask1_2090_30"
              maskUnits="userSpaceOnUse"
              style="mask-type: luminance;"
              width="569"
              x="1"
              y="107"
            >
              <path
                d="M569.616 107H1V462.552H569.616V107Z"
                fill="white"
              />
            </mask>
            <g
              mask="url(#mask1_2090_30)"
            >
              <mask
                height="118"
                id="mask2_2090_30"
                maskUnits="userSpaceOnUse"
                style="mask-type: luminance;"
                width="282"
                x="144"
                y="109"
              >
                <path
                  d="M144.454 109.179H425.761V226.542H144.454V109.179Z"
                  fill="white"
                />
              </mask>
              <g
                mask="url(#mask2_2090_30)"
              >
                <mask
                  height="118"
                  id="mask3_2090_30"
                  maskUnits="userSpaceOnUse"
                  style="mask-type: luminance;"
                  width="282"
                  x="144"
                  y="109"
                >
                  <path
                    d="M425.688 109.179C425.688 173.863 362.762 226.303 285.146 226.303C207.525 226.303 144.6 173.863 144.6 109.179"
                    fill="white"
                  />
                </mask>
                <g
                  mask="url(#mask3_2090_30)"
                >
                  <path
                    d="M568.066 462.375H2.14941V107.343H568.066V462.375Z"
                    fill="currentColor"
                  />
                </g>
              </g>
              <mask
                height="235"
                id="mask4_2090_30"
                maskUnits="userSpaceOnUse"
                style="mask-type: luminance;"
                width="564"
                x="3"
                y="226"
              >
                <path
                  d="M3.98535 226.178H566.235V460.538H3.98535V226.178Z"
                  fill="white"
                />
              </mask>
              <g
                mask="url(#mask4_2090_30)"
              >
                <mask
                  height="235"
                  id="mask5_2090_30"
                  maskUnits="userSpaceOnUse"
                  style="mask-type: luminance;"
                  width="563"
                  x="4"
                  y="226"
                >
                  <path
                    d="M566.23 226.302C566.23 355.665 440.383 460.538 285.147 460.538C129.905 460.538 4.05859 355.665 4.05859 226.302"
                    fill="white"
                  />
                </mask>
                <g
                  mask="url(#mask5_2090_30)"
                >
                  <path
                    d="M568.066 462.374H2.14941V107.343H568.066V462.374Z"
                    fill="currentColor"
                  />
                </g>
              </g>
            </g>
          </g>
        </svg>
      </section>
      <section
        class="space-y-24"
      >
        <article
          class="relative inline-flex flex-col xl:flex-row px-8 sm:px-16"
        >
          <span
            style="font-size: 0px;"
          />
          <h2
            class="font-editorial-new tracking-[0.01em] uppercase opacity-0 w-fit z-10 font-light mt-6 mb-0 sm:mt-6 sm:mb-0 md:mt-6 md:mb-0 absolute right-8 xs:left-[40%] md:left-[400px] text-4xl sm:text-6xl md:text-6xl"
          >
            Arquitecta
          </h2>
          <div
            class="xl:mr-8 shrink-0 h-[400px] w-[230px] xs:w-[320px] sm:w-[400px] relative"
          >
            <img
              alt="Arquitecta"
              class="object-cover"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fabout-us%2Fcoti.png&w=3840&q=75"
              srcset="/_next/image?url=%2Fabout-us%2Fcoti.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Fcoti.png&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
            />
          </div>
          <p
            class="text-base xl:text-lg sm:text-xl max-w-[725px] text-left text-balance mt-[24px] xl:mt-[110px]"
          >
            Después de haber vivido y trabajado cuatro años en Australia y un año en Brasil, pude encontrar mi pasión por la naturaleza. Me gusta pensar cada espacio en relación al exterior y así aprovechar todo lo que nos ofrece en los espacios interiores. Usar materiales nobles, crear espacios funcionales y lo que encuentro fundamental, la Iluminación a lo largo del día con susluces y sombras.
          </p>
        </article>
        <article
          class="relative inline-flex flex-col items-end xl:flex-row-reverse xl:items-start px-8 sm:px-16 "
        >
          <span
            style="font-size: 0px;"
          />
          <h2
            class="font-editorial-new tracking-[0.01em] uppercase opacity-0 w-[200px] sm:w-[310px] md:w-min xl:w-fit flex z-10 font-light mt-6 mb-0 sm:mt-6 sm:mb-0 md:mt-6 md:mb-0 absolute left-8 xs:left-[unset] xs:right-[50%] md:right-[380px] text-4xl sm:text-6xl md:text-6xl"
          >
            Director Creativo
          </h2>
          <div
            class="xl:ml-8 shrink-0 h-[400px] w-[230px] xs:w-[320px] sm:w-[400px] relative"
          >
            <img
              alt="Director Creativo"
              class="object-cover"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fabout-us%2Ffelix.png&w=3840&q=75"
              srcset="/_next/image?url=%2Fabout-us%2Ffelix.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Ffelix.png&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
            />
          </div>
          <p
            class="text-base xl:text-lg sm:text-xl max-w-[725px] text-right text-balance mt-[24px] xl:mt-[110px]"
          >
            Mi interés por la calidez y el paso del tiempo son motivaciones para lograr una visión sensible y natural. Formas, texturas, reflejos, contrastes y movimientos son mis inspiraciones para generar un mundo de sensaciones en el observador. Un ojo agudizado en el detalle.
          </p>
        </article>
      </section>
      <section
        class="lg:hidden space-y-24 flex flex-col items-center"
      >
        <span
          style="font-size: 0px;"
        />
        <h2
          class="font-editorial-new tracking-[0.01em] uppercase mt-12 mb-6 sm:mt-16 sm:mb-9 md:mt-20 md:mb-12 opacity-0 italic text-6xl md:text-6xl"
        >
          Equipo
        </h2>
        <div
          class="p-6 xs:p-8"
        >
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Empresa Constructora
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Civil Construcción SRL
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Paisajismo
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Epifita Arquitectura & Paisaje
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Interiorismo
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Mercedes Navarro
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Proyecto Lumínico
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Alejandra Rodriguez
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Diseñadora
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Trinidad Azpiroz
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Redes
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Lucía de la Torre
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Web
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              Facundo y Matías Marcó del Pont
            </p>
          </div>
          <span
            style="font-size: 0px;"
          />
          <div
            class="space-y-2 mb-14 opacity-0"
          >
            <p
              class="font-editorial-new uppercase font-extralight text-2xl xs:text-3xl sm:text-4xl"
            >
              Marketing y PR
            </p>
            <p
              class="text-xl xs:text-2xl sm:text-3xl"
            >
              María E. Magnolle
            </p>
          </div>
        </div>
        <div
          class="xl:ml-8 shrink-0 h-[400px] w-[90vw] max-w-[800px] relative"
        >
          <img
            alt="Director Creativo"
            class="object-cover"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=%2Fabout-us%2Fequipo_2.png&w=3840&q=75"
            srcset="/_next/image?url=%2Fabout-us%2Fequipo_2.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=3840&q=75 3840w"
            style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
          />
        </div>
      </section>
      <section
        class="hidden lg:flex flex-col p-14"
      >
        <span
          style="font-size: 0px;"
        />
        <h2
          class="font-editorial-new tracking-[0.01em] uppercase sm:mt-16 sm:mb-9 md:mt-20 md:mb-12 opacity-0 w-[600px] lg:w-[1000px] xl:w-[1200px] m-auto italic text-6xl md:text-6xl text-left"
        >
          Equipo
        </h2>
        <div
          class="w-[600px] lg:w-[1000px] xl:w-[1200px] m-auto flex gap-14"
        >
          <div
            class="w-1/2 space-y-14 text-end"
          >
            <div
              class="h-[600px] w-[100%] relative"
            >
              <img
                alt="Director Creativo"
                class="object-cover"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                sizes="100vw"
                src="/_next/image?url=%2Fabout-us%2Fequipo_1.png&w=3840&q=75"
                srcset="/_next/image?url=%2Fabout-us%2Fequipo_1.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Fequipo_1.png&w=3840&q=75 3840w"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
              />
            </div>
            <div
              class="inline-flex flex-col text-right"
            >
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Proyecto Lumínico
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Alejandra Rodriguez
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Diseñadora
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Trinidad Azpiroz
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Redes
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Lucía de la Torre
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Web
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Facundo y Matías Marcó del Pont
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Marketing y PR
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  María E. Magnolle
                </p>
              </div>
            </div>
          </div>
          <div
            class="w-1/2 space-y-14 text-start"
          >
            <div
              class="inline-flex flex-col relative w-[100%]"
            >
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Empresa Constructora
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Civil Construcción SRL
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Paisajismo
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Epifita Arquitectura & Paisaje
                </p>
              </div>
              <span
                style="font-size: 0px;"
              />
              <div
                class="space-y-2 mb-14 opacity-0"
              >
                <p
                  class="font-editorial-new uppercase font-extralight lg:text-3xl xl:text-4xl"
                >
                  Interiorismo
                </p>
                <p
                  class="lg:text-2xl xl:text-3xl"
                >
                  Mercedes Navarro
                </p>
              </div>
              <svg
                class="text-black size-[150px] m-auto absolute bottom-[-30px] right-0"
                fill="none"
                height="24"
                viewBox="0 0 569 569"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <mask
                  height="569"
                  id="mask0_2090_30"
                  maskUnits="userSpaceOnUse"
                  style="mask-type: alpha;"
                  width="569"
                  x="0"
                  y="0"
                >
                  <rect
                    fill="currentColor"
                    height="569"
                    width="569"
                  />
                </mask>
                <g
                  mask="url(#mask0_2090_30)"
                >
                  <mask
                    height="356"
                    id="mask1_2090_30"
                    maskUnits="userSpaceOnUse"
                    style="mask-type: luminance;"
                    width="569"
                    x="1"
                    y="107"
                  >
                    <path
                      d="M569.616 107H1V462.552H569.616V107Z"
                      fill="white"
                    />
                  </mask>
                  <g
                    mask="url(#mask1_2090_30)"
                  >
                    <mask
                      height="118"
                      id="mask2_2090_30"
                      maskUnits="userSpaceOnUse"
                      style="mask-type: luminance;"
                      width="282"
                      x="144"
                      y="109"
                    >
                      <path
                        d="M144.454 109.179H425.761V226.542H144.454V109.179Z"
                        fill="white"
                      />
                    </mask>
                    <g
                      mask="url(#mask2_2090_30)"
                    >
                      <mask
                        height="118"
                        id="mask3_2090_30"
                        maskUnits="userSpaceOnUse"
                        style="mask-type: luminance;"
                        width="282"
                        x="144"
                        y="109"
                      >
                        <path
                          d="M425.688 109.179C425.688 173.863 362.762 226.303 285.146 226.303C207.525 226.303 144.6 173.863 144.6 109.179"
                          fill="white"
                        />
                      </mask>
                      <g
                        mask="url(#mask3_2090_30)"
                      >
                        <path
                          d="M568.066 462.375H2.14941V107.343H568.066V462.375Z"
                          fill="currentColor"
                        />
                      </g>
                    </g>
                    <mask
                      height="235"
                      id="mask4_2090_30"
                      maskUnits="userSpaceOnUse"
                      style="mask-type: luminance;"
                      width="564"
                      x="3"
                      y="226"
                    >
                      <path
                        d="M3.98535 226.178H566.235V460.538H3.98535V226.178Z"
                        fill="white"
                      />
                    </mask>
                    <g
                      mask="url(#mask4_2090_30)"
                    >
                      <mask
                        height="235"
                        id="mask5_2090_30"
                        maskUnits="userSpaceOnUse"
                        style="mask-type: luminance;"
                        width="563"
                        x="4"
                        y="226"
                      >
                        <path
                          d="M566.23 226.302C566.23 355.665 440.383 460.538 285.147 460.538C129.905 460.538 4.05859 355.665 4.05859 226.302"
                          fill="white"
                        />
                      </mask>
                      <g
                        mask="url(#mask5_2090_30)"
                      >
                        <path
                          d="M568.066 462.374H2.14941V107.343H568.066V462.374Z"
                          fill="currentColor"
                        />
                      </g>
                    </g>
                  </g>
                </g>
              </svg>
            </div>
            <div
              class="h-[900px] w-[100%] relative"
            >
              <img
                alt="Director Creativo"
                class="object-cover"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                sizes="100vw"
                src="/_next/image?url=%2Fabout-us%2Fequipo_2.png&w=3840&q=75"
                srcset="/_next/image?url=%2Fabout-us%2Fequipo_2.png&w=640&q=75 640w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=750&q=75 750w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=828&q=75 828w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1080&q=75 1080w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1200&q=75 1200w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=1920&q=75 1920w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=2048&q=75 2048w, /_next/image?url=%2Fabout-us%2Fequipo_2.png&w=3840&q=75 3840w"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
              />
            </div>
          </div>
        </div>
      </section>
    </div>
  </main>
</div>
`;
